src/main/java/pl/sda/weather/service/ReadWeathers.java

```diff
@@ -0,0 +1,17 @@
+package pl.sda.weather.service;
+
+import pl.sda.weather.model.LocationModel;
+import pl.sda.weather.model.Weather;
+import pl.sda.weather.model.WeatherLine;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ReadWeathers {
+
+    Map<String,Weather> mapWeather();
```

Comment:
getWeatherMap() would be better (mapWeather means - do mapping on Weather object)